.travis.yml

```diff
@@ -16,14 +16,14 @@ after_success:
     sonar-scanner;
    fi
 deploy:
-- provider: releases
-  api_key:  ${DEPLOY_TOKEN}
-  file:
-    - "vscode-apache-camel-0.0.2.vsix"
-  skip_cleanup: true
-  overwrite: true
-  on:
-    branch:  master
+#- provider: releases
```

Comment:
why not completely removing it instead of commenting it?
have you found back the right to edit the marketplace entry?
the Install manually from the readme needs to be removed too